### DISCUSSION TITLE
Quering views returns bad data

### DISCUSSION THREAD

**Discussion by g13n4** [fastapi/sqlmodel#827] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from sqlmodel import SQLModel

class PerformanceViewBase(SQLModel):
    __table_args__ = {'info': {
        'is_view': True
    }}
   # etc, it's very big
    pass

class PerformanceWindowBase(SQLModel):
    l2: condecimal(max_digits=10, decimal_places=2) = Field(default=None, nullable=True)
    l4: condecimal(max_digits=10, decimal_places=2) = Field(default=None, nullable=True)
    l6: condecimal(max_digits=10, decimal_places=2) = Field(default=None, nullable=True)
    l8: condecimal(max_digits=10, decimal_places=2) = Field(default=None, nullable=True)
    l10: condecimal(max_digits=10, decimal_places=2) = Field(default=None, nullable=True)
    ltotal: condecimal(max_digits=10, decimal_places=2) = Field(default=None, nullable=True)

    g15: condecimal(max_digits=10, decimal_places=2) = Field(default=None, nullable=True)
    g30: condecimal(max_digits=10, decimal_places=2) = Field(default=None, nullable=True)
    g45: condecimal(max_digits=10, decimal_places=2) = Field(default=None, nullable=True)
    g60: condecimal(max_digits=10, decimal_places=2) = Field(default=None, nullable=True)
    g60plus: condecimal(max_digits=10, decimal_places=2) = Field(default=None, nullable=True)
    gtotal: condecimal(max_digits=10, decimal_places=2) = Field(default=None, nullable=True)


class PerformanceWindowView(PerformanceViewBase, PerformanceWindowBase, table=True):
    __tablename__ = 'performance_window_view'


clauses = [PerformanceWindowView.match_id == 7258005870,
PerformanceWindowView.data_type_id == 1,
PerformanceWindowView.is_comparison == False]

query = db.exec(select(PerformanceWindowView)
                    .where(and_(*clauses))))
```

### Description

So I've have created class objects, queried them and everything works fine. I created a view in postgres and can query it in pgadmin (using sqlalchemy's execute text too) without any problems but when I do using the orm model it just return a copy of the first row N amount of times instead of the real data.

Here is an example of me querying data in pgadmin and using sqlmodel's orm
![Screenshot_20240301_215213](https://github.com/tiangolo/sqlmodel/assets/24457993/2f718f3b-2a5c-43f6-855e-00dfbf2001f5)

### Operating System

Linux

### Operating System Details

Manjaro linux, postgres 15.3

### SQLModel Version

0.0.16

### Python Version

3.11

### Additional Context

I open to any ideas how to circumvent this. My data is normalised so I can't really do without a view

**Comment by SonOfLilit** [comment]
Why are you inheriting from `PerformanceWindowBase`?

If you could write a failing test / otherwise provide something a dev might be able to reproduce on their machine, that would go 90% to identifying and solving the problem.

**Reply by g13n4** [reply]
The data is proprietary so I would prefer not to show the models. They are absolutely fine. I just inherit from one to make a more query friendly model. It's more of a generalised question whether querying views work for anyone else, especially for people with billions and more rows. I tried using execute instead of exec and the result was the same so maybe it's sql-alchemy's bug

**Comment by YuriiMotov** [comment]
MRE is needed
